### PR TITLE
Readmes and reorganization

### PR DESCRIPTION
**Pull request by nabalone**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/15)

### CODE REVIEW

File: components/language-chooser/common/find-language/README.md [hunk 1]
```diff
@@ -1,2 +1,160 @@
...
+
+## About
+
+This project was initially developed for use in [BloomDesktop](https://github.com/BloomBooks/BloomDesktop).
+
+This component contains the logic for fuzzy-searching languages, designed to be used by frontend language choosers. The language database is based on [langtags.json](https://github.com/silnrsi/langtags) and also references [langtags.txt](https://github.com/silnrsi/langtags/blob/master/doc/tagging.md#langtagstxt). We use [fuse.js](https://fusejs.io/) for fuzzy-searching.
+
+It also contains various utilities for working with language tags and language information.
+
+<!-- TODO move to usage section? -->
```
Comment (nabalone) [inline components/language-chooser/common/find-language/README.md:14]: Not sure which section Search Result Modification should go in

**Comment by nabalone** [comment]
Ready for review. Still some TODOs/sections to fill in but ready for feedback

**Review by andrew-polk — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/15)* status: all files reviewed, 1 unresolved discussion (waiting on @nabalone)

___
*[`components/language-chooser/common/find-language/README.md` line 14 at r1](https://reviewable.io/reviews/sillsdev/EthnoLib/15#gh-1792633742:-O8lpRFk0a1lmu4ER2Ui:b-sagspi) ([raw file](https://github.com/sillsdev/EthnoLib/blob/0ab58e3ca5092055cd51fcc0e0cebeccc81dad1a/components/language-chooser/common/find-language/README.md#L14)):*
<details><summary><i>Previously, nabalone (Noel) wrote…</i></summary><blockquote>

Not sure which section Search Result Modification should go in
</blockquote></details>

Probably usage. Thanks.

**Review by andrew-polk — CHANGES_REQUESTED** [review]
> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/15)* status: all files reviewed, 1 unresolved discussion (waiting on @nabalone)

File: components/language-chooser/common/find-language/scripts.json
Comment (nabalone) [inline components/language-chooser/common/find-language/scripts.json:1]: This file is unused and was in the repo by mistake

**Comment by nabalone** [comment]
Ready for review

**Review by nabalone — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/15)* status: 1 of 14 files reviewed, 2 unresolved discussions (waiting on @andrew-polk)

___
*[`components/language-chooser/common/find-language/README.md` line 14 at r1](https://reviewable.io/reviews/sillsdev/EthnoLib/15#gh-1792633742:-O8sIfIL7CTE_AA0nDIw:b-896fix) ([raw file](https://github.com/sillsdev/EthnoLib/blob/0ab58e3ca5092055cd51fcc0e0cebeccc81dad1a/components/language-chooser/common/find-language/README.md#L14)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

Probably usage. Thanks.
</blockquote></details>

Done.

**Review by andrew-polk — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/15)* status: all files reviewed, 1 unresolved discussion

**Pull request merged**